src/components/OrganizationsTable.tsx

```diff
@@ -1,45 +1,58 @@
 import React from "react";
 import gql from "graphql-tag";
-import { Table } from "antd";
+import { Table } from 'antd';
 import { useQuery } from "react-apollo";
 
+
 const columns = [
   {
     name: "Organization",
     dataIndex: "orgFullName",
     key: "orgFullName",
+    onFilter: (value: string, record: IOrganization) => record.orgFullName.indexOf(value) === 0,
+    sorter: (a: IOrganization, b: IOrganization) => a.orgFullName.localeCompare(b.orgFullName)
   },
   {
     title: "Trials",
     dataIndex: "totalCount",
     key: "totalCount",
+    filters: [{text: '>20', value: '20'}, {text: '>50', value: '50'}],
+    onFilter: (value: number, record: IOrganization) => (record.totalCount - value) > 0,
+// This inexplicably doesn't work!!!!
+//     onFilter: (value: number, record: IOrganization) => record.totalCount > value,
+    sorter: (a: IOrganization, b: IOrganization) => a.totalCount - b.totalCount,
   },
   {
     title: "Results on time (%)",
     dataIndex: "onTimeFrac",
     key: "onTimeFrac",
+    sorter: (a: IOrganization, b: IOrganization) => a.onTimeFrac - b.onTimeFrac,
```

Comment:
This is nice, although I'd vote for doing sorting on the server side and using pagination. It's faster :)

Here's an example:
```
query MyQuery {
  allOrganizations(first: 10, orderBy: MISSING_COUNT_DESC) {
    nodes {
      orgFullName
      orgClass
      shouldHaveResultsCount
      missingFrac
      onTimeFrac
      lateFrac
    }
    pageInfo {
      endCursor
      hasNextPage
      hasPreviousPage
      startCursor
    }
  }
}
```

Reply:
Hey, so the idea here is that this is just an additional level of flexibility on top of the sorting done in the request. The data populates using the default, but now, you can modify the table sorting by clicking the headers (and add an optional filter) without re-requesting the data
![image](https://user-images.githubusercontent.com/5759196/72221779-69994000-3523-11ea-8d1e-497b623d6ee9.png)

Reply:
oh, but are you suggesting that we should really just be requesting the first page of data rather than pulling everything and populating the full data into the table? I think that's makes sense as an optimization but I'm not sure how to integrate that with the current Table framework so maybe could be a followon

Reply:
Yep, exactly! To request just the first page of data and still support sorting, clicking the column headers needs to be hooked up to making new `gql` requests. Additionally, the backend returns meta info like `totalCount` which should allow you to draw the table (with correct page count) even if you have no more than the current page's worth of results.

That said, I agree with you. Feel free to merge this PR, and we can add server-side pagination later.

Reply:
FYI I added this as an issue https://github.com/tomquisel/clinical-trials-report-frontend/issues/11